crates/ere-openvm/src/lib.rs

```diff
@@ -149,34 +149,63 @@ impl EreOpenVM {
             agg_pk,
             agg_vk,
             app_commit,
-            _resource,
+            resource,
         })
     }
 
-    fn sdk(&self) -> Result<Sdk, CommonError> {
-        let sdk =
-            Sdk::new_without_transpiler(self.app_config.clone()).map_err(CommonError::SdkInit)?;
+    fn cpu_sdk(&self) -> Result<CpuSdk, CommonError> {
+        let sdk = CpuSdk::new_without_transpiler(self.app_config.clone())
```

Comment:
Is there any relevant implication of using `CpuSDK::new_without_transpiler` here compared to the `CpuSDK::new` in the ctor above?